### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by ajay-keelu**
Task completed

### CODE REVIEW

File: css/main.css
Comment (senasowseelya) [inline css/main.css:1]: Rename this to index.css

File: images/Group 11.png
Comment (senasowseelya) [inline images/Group 11.png:1]: Rename all the images accordingly , the names are not consistent.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css" integrity="sha512-DTOQO9RWCH3ppGqcWaEA1BIZOC6xxalwEsw9c2QQeAIftl+Vegovlnee1c9QX4TctnWMn13TZye+giMm8e2LwA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  </head>
+  <body>
+    <!-- header with nav links -->
+    <header id="header">
+      <div class="logo">
+        <a href="#"><img src="./images/Decore.png" alt="logo image" /></a>
+      </div>
+      <div class="navitems">
```
Comment (senasowseelya) [inline index.html:16]: follow kebab case  for css class names (use nav-items)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+            </div>
+          </div>
+          <div class="item-container">
+            <div class="icon">
+              <img src="./images/Group 12.png" alt="" />
+            </div>
+            <div class="description">
+              <div class="heading">Make Payment</div>
+              <div class="head-description">
+                Lorem ipsum dolor sit amet, consectetur <br/> adipisicing elit. Atque,
```
Comment (senasowseelya) [inline index.html:168]: Replace tis random text with actual content

File: images/Group 11.png
Comment (ajay-keelu) [reply images/Group 11.png:1]: ok ,I'll rename all

File: css/main.css
Comment (ajay-keelu) [reply css/main.css:1]: ok

File: index.html (hunk 2 above)
Comment (ajay-keelu) [reply index.html:168]: i've written same text given in the image
Home Page

File: index.html [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+        <a href="#"><img src="./images/Decore.png" alt="logo image" /></a>
+      </div>
+      <div class="navitems">
+        <ul>
+          <li>Destinations</li>
+          <li>Hotels</li>
+          <li>Flights</li>
+          <li>Bookings</li>
+          <li>Login</li>
+          <li><button id="Signup">Sign up</button></li>
```
Comment (senasowseelya) [inline index.html:23]: The background color is not matching for the button with the mockup buttton

File: index.html (hunk 1 above)
Comment (ajay-keelu) [reply index.html:16]: ok

**Pull request merged**